vars/buildDockerAndPublishImage.groovy

```diff
@@ -21,16 +21,6 @@ def makecall(String action, String imageDeployName, String targetOperationSystem
       sh "make bake-$action"
     }
   } else {
-    if (action == 'deploy') {
-      if (env.TAG_NAME) {
-        // User could specify a tag in the image name. In that case the git tag is appended. Otherwise the docker tag is set to the git tag.
-        if (imageDeployName.contains(':')) {
-          imageDeployName += "-${env.TAG_NAME}"
-        } else {
-          imageDeployName += ":${env.TAG_NAME}"
-        }
-      }
-    }
```

Comment:
This code does not seem related to the topic and should stay. It is used by https://github.com/jenkins-infra/docker-inbound-agents (worth a comment to add though)